.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "0.10"
+sudo: false
+before_install:
+  - npm install -g npm@2.14.9
```

Comment:
small thing (and probably not worth changing), but I think we're using 2.13.5 mostly these days (in fhcap at least)